sc.yaml

```diff
@@ -0,0 +1,15 @@
+# yaml file for creating StorageClass
+#
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  annotations:
+    storageclass.kubernetes.io/is-default-class: "true"
+  name: test
```

Comment:
Let's use full name `storage_class.yaml`.  Can you change `test` to `default`?

BTW, have you validate the yaml by submitting to cluster?

Reply:
OK I'll change the sc.yaml to storage_class.yaml.
I've validated the yaml by submitting to cluster, it works.